raiden/network/transport/matrix/utils.py

```diff
@@ -18,11 +18,14 @@
 from raiden.network.utils import get_http_rtt
 from raiden.utils.signer import Signer, recover
 from raiden.utils.typing import Address
+from raiden_contracts.constants import ID_TO_NETWORKNAME
 
 log = structlog.get_logger(__name__)
 
 JOIN_RETRIES = 5
 userid_re = re.compile(r'^@(0x[0-9a-f]{40})(?:\.[0-9a-f]{8})?(?::.+)?$')
+room_name_separator = '_'
+room_name_prefix = 'raiden'
```

Comment:
I'd say those should be upper case constants

Reply:
Done, thank you